### ISSUE TITLE
Meanjs.org, roadmap and the path going forward

### ISSUE THREAD

**Issue by SOSANA** [meanjs/mean#499]
Hi guy's,

Haven't seen a blog post in a while on meanjs.org and we abondon using trello https://trello.com/b/5FAPcode/mean-js or https://trello.com/b/auT2l1Ff/mean-js-0-4-x

I know @ilanbiala is contrbuting like a madman trying to push v4.0 obviously he needs our help. But from watching the repo have an idea where were going, but we should have a roadmap,a blog post and change log update, some where we can see updates  or goals and other issues that need help from the community to contribute like we had for a short time on trello. We need to fire up some updated communication for the rest of community to see.

On as side note just my crazy thoughts and 2 cents...Meanjs v5.0 for future roadmap Angular 2.0.?  crazy to think but angular release coming sooner than we think.

any thoughts @roieki @amoshaviv @ilanbiala @lirantal ??

**Comment by dotch** [comment]
Some public discussion on the road map would be really nice.

Things like switching to the new router with angular 1.4 and completely moving to gulp for easier maintenance should be discussed after the 0.4.0 release.

**Comment by SOSANA** [comment]
(crickets) - @roieki @amoshaviv @ilanbiala @lirantal

**Comment by ilanbiala** [comment]
@SOSANA I can tell you what I'd like to see and what's going on, but ultimately I need some input from the rest of the team and some of the PRs are waiting on the user for a rebase or a new PR against 0.4.0 instead of master.

**Comment by SOSANA** [comment]
@ilanbiala thanks for responding back. Lets here what you would like to see. I just want some communication on the future of the project.

**Comment by ilanbiala** [comment]
@SOSANA the things I would expect of 0.4.0 can be found in the milestone. Anything else I would think could be added after without breaking any of the code or requiring too much time now, so I left that in the 0.4.x milestone. It would also be great to get some more contributors because I'd really like to see pluggable systems for pubsub and other popular but secondary pieces to the puzzle. I think if we can expose a nice easy way to allow multiple pubsub implementations to be used or support that in the generator, that would be best. That probably won't happen in 0.4.x though, and we'll see once 0.4.0 is released what the community is looking for in future versions.

**Comment by codydaig** [comment]
Can this be closed now that we have a task list for 0.4.0 open and another issue open for the 0.5.0 roadmap?
